cdp/wallet_address.py

```diff
@@ -221,6 +222,87 @@ def sign_payload(self, unsigned_payload: str) -> PayloadSignature:
             signature=signature,
         )
 
+    def deploy_token(self, name: str, symbol: str, total_supply: str) -> SmartContract:
+        """Deploy a token smart contract.
+
+        Args:
+            name (str): The name of the token.
+            symbol (str): The symbol of the token.
+            total_supply (Union[int, float, Decimal, str]): The total supply of the token.
```

Comment:
The type hint for `total_supply` should be `Number | Decimal | str`

Can we update these across all references?

Reply:
good catch updating the docs to say string only